tests/playlists.spec.js

```diff
@@ -34,6 +34,37 @@ describe('Test Playlist Endpoints', () => {
         .send({title: 'Different Playlist Name'})
       expect(res.body[0].id).toBe(oldPlaylist[0].id)
       expect(res.body[0].title).toBe('Different Playlist Name')
+      expect(res.body[0].title).toBe('Different Playlist Name')
+    })
+  })
+
+  describe('Get Playlist', () => {
+    it('Gets a list of all playlists', async () => {
+    const playlist1 = { title: 'Working Out' }
+    const playlist2 = { title: 'Driving' }
+    const db1 = await database('playlists').returning(['id', 'title']).insert(playlist1).then(result => result)
+    const db2 = await database('playlists').returning(['id', 'title']).insert(playlist2).then(result => result)
+    const res = await request(app)
+    .get('/api/v1/playlists')
+
+    expect(res.body[0].title).toBe('Working Out')
+    expect(res.body[0].id).toBe(db1[0].id)
+    expect(res.body[1].title).toBe('Driving')
+    expect(res.body[1].id).toBe(db2[0].id)
+
     })
   })
+  
+  describe('Delete Playlist', () => {
+    it('Deletes a single playlist by id', async () => {
+        const newPlaylist = { title: 'At Home' }
+        const req = await database('playlists')
+         .returning(['id', 'title'])
+         .insert(newPlaylist)
+        const res = await request(app)
+         .delete(`/api/v1/playlists/${req[0].id}`)
+
+        expect(res.statusCode).toBe(204)
```

Comment:
I think this is great just testing for the status code. If there were any errors with the delete we wouldn't get a 204 back.